index.html

```diff
@@ -1,14 +1,67 @@
+        title="Pic3" style="width: 200px; height: 250px;" />
+      
+        <a href='learn-more.html' class="button">Learn More</a>
+    </div>
+  </div>
+  <div class="container2">
+      <div class="item quote-box">"Help me, Obi-Wan Kenobi. You’re my only hope.” - Leia Organa</div>
+  </div> 

```

Comment:
This is very close to living on the page how it should be, Take a look at the CSS for these sections. It is a simple fix to make this stretch across the whole screen